packages/mobile/src/screens/favorites-screen/OfflineContentBanner.tsx

```diff
@@ -43,7 +44,7 @@ export const OfflineContentBanner = () => {
   const isReachable = useSelector(getIsReachable)
   if (!isOfflineModeEnabled || isReachable) return null
   return (
-    <View>
+    <Animated.View exiting={SlideOutUp.duration(1000)}>
```

Comment:
should it slide in too?

Reply:
So when experimenting it slides in due to the surrounding layout transition, but needed this for the disappearance to look smooth. lets def play with it on device when up to make sure it looks clean

Reply:
ah, okay cool